lib/npm-installation-manager.ts

```diff
@@ -20,15 +20,22 @@ export class NpmInstallationManager implements INpmInstallationManager {
 	}
 
 	public async getLatestCompatibleVersion(packageName: string): Promise<string> {
+		const configVersion = this.$staticConfig.version;
+		const isPreReleaseVersion = (<any>semver).prerelease(configVersion) !== null;
```

Comment:
I do believe our definitions for `semver` already include a definition for `prerelease` so you can lose the `<any>` cast